ansible_ai_connect/ai/api/model_pipelines/http/pipelines.py

```diff
@@ -131,6 +131,7 @@ def invoke(self, params: ChatBotParameters) -> ChatBotResponse:
         conversation_id = params.conversation_id
         provider = params.provider
         model_id = params.model_id
+        system_prompt = params.system_prompt
```

Comment:
Changes in this file (`pipelines.py`) are the fix for #1480, which added `system_prompt` as a parameter of `/chat` endpoint, but it was not sent to Chatbot serivice.